Homework_10_Messenger/Messenger_program/src/Main.java

```diff
@@ -0,0 +1,74 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        User alex = new User("Alex");
+        User ira = new User("Ira");
+        User ola = new User("Olia");
+        User marta = new User ("Marta");
+
+
+        alex.addContact(ira);
+        alex.addContact(ola);
+        ira.addContact(alex);
+        ola.addContact(alex);
+        marta.addContact(ola);
+        marta.addContact(alex);
+        ola.addContact(marta);
+
+
+        alex.sendMessage(ira, "Hi, have you done your java homework?");
+        ira.sendMessage(alex, " Ні Alex! It's so complicated, I've been studying it for so long. I haven't done it yet. And you?");
+        ola.sendMessage(alex, "Hi Alex! Tell me how to write this messenger program");
+        marta.sendMessage(ola, "Hi Olia, I heard that you have some problems with homework");
+        ola.sendMessage(marta, "Hi, I give up with this  messenger program!");
+        marta.sendMessage(ola, "Try to ask AI, it will give you some advice.");
+        alex.sendMessage(ira, "It was a piece of cake for me!");
+
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter your username (choose one : alex, marta, ira, olia) : ");
+        String userName = scanner.nextLine();
+
+        User currentUser = getUserByName(userName, alex, marta, ira, ola);
+
+        if (currentUser != null) {
+
+            System.out.println("Contacts for " + currentUser.getUserName() + ":");
+            for (UserInterface contact : currentUser.getContacts()) {
+                System.out.println(contact.getUserName());
+            }
+
+
+            System.out.println("\nMessages for " + currentUser.getUserName() + ":");
+            for (MessageInterface message : currentUser.getMessages()) {
```

Comment:
Not a mistake, but looks like a good candidate to be extracted into `.printAllMessages()` method